src/main/java/net/sacredlabyrinth/phaed/simpleclans/commands/clan/ClanCommands.java

```diff
@@ -59,7 +56,7 @@ public void startWar(Player player, ClanPlayer cp, Clan issuerClan, @Name("clan"
                 ChatBlock.sendMessage(player, RED + lang("at.least.one.leader.accept.the.alliance", player));
             }
         } else {
-            ChatBlock.sendMessage(player, RED + lang("clans.already.at.war", player));
+            startWar(player, cp, issuerClan, other);
```

Comment:
https://github.com/RoinujNosde/SimpleClans/blob/master/src/main/java/net/sacredlabyrinth/phaed/simpleclans/managers/RequestManager.java#L353-L362
This could be extracted to https://github.com/RoinujNosde/SimpleClans/blob/d892b031422fbbc5aa2c21c60b34652451a14ac6/src/main/java/net/sacredlabyrinth/phaed/simpleclans/managers/ProtectionManager.java#L124
and we would call ProtectionManager#addWar here